relationships/candidates/AWSMQQUEUE.yml

```diff
@@ -0,0 +1,18 @@
+category: AWSMQQUEUE
```

Comment:
Shouldn't we generalize this to AmazonMQ/ActiveMQ/RabbitMQ, as we did for ElastiCache?